libevm/VM.cpp

```diff
@@ -44,6 +44,40 @@ template <class S> S modWorkaround(S const& _a, S const& _b)
 
 
 //
+// for decoding destinations of JUMPTO, JUMPV, JUMPSUB and JUMPSUBV
+//
+
+uint64_t VM::decode_jump_dest(const byte* const _code, uint64_t& _pc)
+{
+	// turn 4 MSB-first bytes in the code into a native-order integer
+	uint64_t dest      = _code[_pc++];
+	dest = (dest << 8) | _code[_pc++];
+	dest = (dest << 8) | _code[_pc++];
+	dest = (dest << 8) | _code[_pc++];
+	return dest;
+}
+
+uint64_t VM::decode_jumpv_dest(const byte* const _code, uint64_t& _pc, u256*& _sp)
```

Comment:
Style: same as the previous function.
Also, `u256*&` is a bit scary. Can you add a comment about how it behave?